README.md

```diff
@@ -1,2 +1,84 @@
-# math-magicians
-This is math magicians project
+# Math-Magicians
+
+- This is the first project of the Math Magicians application. You will set up the environment and tools needed to develop a React application. In the following projects, you will develop the actual application.
+
+## Built With
+
+- React
+- HTML
+- CSS
+- JavaScript
+
+## Getting Started
+- This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- Clone this repo to your local machine using `
+- cd into the folder and run `npm install` to install all the dependencies.
+- Run `npm start` to start the app.
+- Open `http://localhost:3000` to view it in the browser.
+- The page will reload if you make edits.
+- You will also see any lint errors in the console.
+- Run `npm run build` to build the app for production to the `build` folder.
+- It correctly bundles React in production mode and optimizes the build for the best performance.
+- The build is minified and the filenames include the hashes.
+- Your app is ready to be deployed!
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+
+## prerequisites
+- must have git installed.
+- must have vscode installed or any other IDE.
+- must installed linters in the project
+- must have node installed.
+
+## setup
+- git clone git@github.com:demix007/math-magicians.git
+- cd math-magicians
+- npm install
+- npm start
+
+## Authors
+
+👤 *Author Ayodeji Abidemi*
+
+- GitHub: [@githubhandle]([https://github.com/githubhandle](https://github.com/demix007))
+- Twitter: [@twitterhandle][(https://twitter.com/twitterhandle)](https://twitter.com/dat_dope_demix)
+- LinkedIn: [LinkedIn][(https://linkedin.com/in/linkedinhandle)](https://linkedin.com/in/ayodeji-abidemi-b76935218)
+
+
+
+## LIVE DEMO
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b 'branchname`)
+3. Commit your Changes (`git commit -m 'Add some branchname'`)
+4. Push to the Branch (`git push origin branchname`)
+5. Open a Pull Request
+
+Feel free to check the [issues page](https://github.com/demix007/math-magicians/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+* [Choose an Open Source License](https://choosealicense.com)
+* [GitHub Emoji Cheat Sheet](https://www.webpagefx.com/tools/emoji-cheat-sheet)
+* [Malven's Flexbox Cheatsheet](https://flexbox.malven.co/)
+* [Malven's Grid Cheatsheet](https://grid.malven.co/)
+* [Img Shields](https://shields.io)
+* [GitHub Pages](https://pages.github.com)
+* [Font Awesome](https://fontawesome.com)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Your README looks professional, and Great job adding LICENSE 🙌 

- [OPTIONAL] I noticed that the MIT Link is broken in your README file, so I suggest fixing it 
by changing the path from `[MIT](./MIT.md)` to `[MIT](./LICENSE)` 💯👍
> or rename the LICENSE file to MIT.md 😅

Reply:
# Hello @Bondok6   👋🏼,   

Thank you for taking out time to review and approve my PR. I have made the suggested optional change, thank you for calling my attention to it. You are amazing!!! 😎